configs/beit/upernet_beit_base_8x2_640x640_160k_ade20k.py

```diff
@@ -0,0 +1,30 @@
+_base_ = [
```

Comment:
configs/beit/upernet_beit_base_8x2_640x640_160k_ade20k.py -->
configs/beit/upernet_beit-base_8x2_640x640_160k_ade20k.py

The same for others.